catroid/src/main/res/values/strings.xml

```diff
@@ -1905,6 +1905,7 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_list_dialog_title">New list</string>
     <string name="formula_editor_discard_changes_dialog_message">Do you want to save your changes?</string>
     <string name="formula_editor_discard_changes_dialog_title">Save changes?</string>
+    <string name="formula_editor_save_changes_dialog_title">Save changes?</string>
```

Comment:
please just use the old string which is 
<string name="formula_editor_discard_changes_dialog_title">Save changes?</string>
, no need to have a new one.